### DISCUSSION TITLE
Update Video?

### DISCUSSION THREAD

**Discussion by TechStudent10** [ThioJoe/YT-Spammer-Purge#90] [Q&A] [answered]
A lot of things have changed since @ThioJoe made his initial video. Maybe it's time for an update video to show off the colors, new modes, etc.

**Comment by KendallDoesCoding** [comment]
Well, I don't think he should as he has other content to make. If people are interested in getting a update, they can visit the github releases page.

**Reply by TechStudent10** [reply]
Well yeah, but you have to think, how many YouTubers are willing to go through EVERY GitHub release since the version he used in his original video. And, well, yes, they can look through the readme, but if there is an option, people would rather watch a video. And even stuff like NUKE MODE, it isn't even documented in the README (at least I think so). So, an update video would be nice if not necessary.

**Reply by KendallDoesCoding** [reply]
True, that's some facts

**Comment by ThioJoe** [comment, marked as answer]
Yea I was planning on making one early this week, but YouTube's freakin API is literally broken since Friday and won't delete comments. They still haven't fixed it.
https://issuetracker.google.com/u/1/issues/206873638

**Reply by TechStudent10** [reply]
oh ok. I tried starring this but my request got "throttled". I'll try again today.

**Reply by TechStudent10** [reply]
*the other one

**Reply by KendallDoesCoding** [reply]
> Yea I was planning on making one early this week, but YouTube's freakin API is literally broken since Friday and won't delete comments. They still haven't fixed it. https://issuetracker.google.com/u/1/issues/206873638

@ThioJoe I believe this issue tracker thing has been fixed now. Please check it out!

**Reply by TechStudent10** [reply]
Update video time!

**Reply by ThioJoe** [reply]
yea I know, I want to add a couple little features first like being able to custom search both usernames and comments simultaneously.

**Reply by TechStudent10** [reply]
@ThioJoe sounds good!

**Comment by KendallDoesCoding** [comment]
Same!

**Comment by KendallDoesCoding** [comment]
sounds good @ThioJoe
